### DISCUSSION TITLE
About New OCR model’s improvement

### DISCUSSION THREAD

**Discussion by cyhasuka** [infiniflow/ragflow#3213] [Q&A] [answered]
Concerned that yesterday huggingface had an update on the latest OCR model.

I performed a comparison test, including file slicing, GENERAL mode, and MANUAL mode embedding. But I am puzzled that I didn't find any change in the OCR results after updating the model, may I ask what are the improvements in the updated model? 

Since OCR is critical for document recognition accuracy, are there any plans for further optimization? Thx.

**Comment by KevinHuSh** [comment]
Misoperation, ignore it please.
Yes, we're researching.

**Comment by yingfeng** [comment, marked as answer]
New document layout analysis model is introduced, see #4054

**Reply by cyhasuka** [reply]
That's Great!
